### ISSUE TITLE
Relative path must be derived from curent project folder 

### ISSUE THREAD

**Issue by dim-37** [fsprojects/SwaggerProvider#164]
![image](https://user-images.githubusercontent.com/29534485/98292019-b1a6b900-1fb4-11eb-9683-eba558ab7de3.png)
![image](https://user-images.githubusercontent.com/29534485/98292155-f0d50a00-1fb4-11eb-9844-b9b6798193e0.png)

**Comment by sergey-tihon** [comment]
It is not easy reliable detect current project folder from TP runtime.
In F# we usually use `__SOURCE_DIRECTORY__` [predefined identifiers](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/source-line-file-path-identifiers) for that

```fsharp
let [<Literal>] Schema = __SOURCE_DIRECTORY__ + "../Oiler/schema.json"
type OilChange = OpenApiClientProvider<Schema>
```

**Comment by petrkoutnycz** [comment]
@sergey-tihon Thanks for the tip, I had no idea there was something like that.

**Comment by xperiandri** [comment]
How is it different from what FSharp.Data.GraphQL does?
Is it different for generative and erased type providers?

**Comment by sergey-tihon** [comment]
> Is it different for generative and erased type providers?
There should not be difference between generative and erased type providers.

> Assembly.GetExecutingAssembly().Location
Executing assembly for design-time component of TP is the IDE or compiler

The path of current project should somewhere in [TypeProviderConfig](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-compilerservices-typeproviderconfig.html#ResolutionFolder) that we receive in [the ctor](https://github.com/fsprojects/SwaggerProvider/blob/master/src/SwaggerProvider.DesignTime/Provider.OpenApiClient.fs#L16) - it is most likely `ResolutionFolder` property.

**Comment by xperiandri** [comment]
Fixed in #171
